tools/rtpsrelay/RelayHandler.h

```diff
@@ -83,6 +88,7 @@ class GuidAddrSet {
               RelayStatisticsReporter& relay_stats_reporter)
     : config_(config)
     , relay_stats_reporter_(relay_stats_reporter)
+    , expected_discovery_time_(config.lifespan() / 10)
```

Comment:
The errors for this look like an oversight on my part. There are multiplication operators for `TimeDuration`, but not ones for division. So you could add them or do:

```suggestion
    , expected_discovery_time_(config.lifespan() * 0.1)
```